README.rst

```diff
@@ -127,7 +127,7 @@ can use a search:
     >>> Particle.from_pdgid(211)
     <Particle: pdgid=211, fullname='pi+', mass=139.57061 ± 0.00024 MeV>
     >>>
-    >>> Particle.from_search_list('pi')[0]
```

Comment:
By removing this line you mean that this simple shortcut with just a string as name search won't be possible anymore? Not the end of the world but a bit of a shame IMO.

Reply:
This has become `name='pi'`, but that's exact match only. So the proper replacement for this is:

```Python
Particle.from_search_list(name = lambda x: 'pi' in x)[0]
```

Would it make sense to use the first argument as a fuzzy name search if it is not callable? Then this shortcut would return.

Reply:
I'm confused now since from_search_list is different from from_search in that the former returns a list of matches. I guess the answer to your question is yes - if I understand correctly.

Reply:
Thanks for that.